src/components/card/card.tsx

```diff
@@ -10,7 +10,7 @@ export interface CardBaseProps extends HTMLAttributes<HTMLOrSVGElement> {
 export interface CardHeaderIconProps extends HTMLAttributes<HTMLOrSVGElement> {
   className?: string
   children?: React.ReactNode
-  onClick?: () => void
+  onClick?: (e: React.MouseEvent<HTMLAnchorElement>) => void
```

Comment:
Will this onclick only ever be added to an anchor tag?

Reply:
Yes, see line 18 and also the collapse examples of it in action